cmake/SpectreSetupPythonPackage.cmake

```diff
@@ -35,8 +35,26 @@ set(PYTHON_EXE_COMMAND "-m spectre")
 set(PYTHON_EXEC_ENV_VARS "")
 if(BUILD_PYTHON_BINDINGS AND "${JEMALLOC_LIB_TYPE}" STREQUAL SHARED)
   string(APPEND PYTHON_EXEC_ENV_VARS
-    " LD_PRELOAD=\${LD_PRELOAD}\${LD_PRELOAD:+:}${JEMALLOC_LIBRARIES}")
+    " \\\nLD_PRELOAD=\${LD_PRELOAD}\${LD_PRELOAD:+:}${JEMALLOC_LIBRARIES}")
 endif()
+# TODO: it would be good to figure out if there's a more general way to do this,
+#       and for example if we need to do it for _all_ dependencies. It probably
+#       wouldn't hurt to do it for more dependencies, e.g. MPI.
```

Comment:
One issue I have encountered is that the env variables defined here also propagate to the environment that submits jobs through the CLI, and hence to the job itself. We just need to make sure that's ok (and add a comment here to remember this potential issue in the future).